api/src/main/java/uk/gov/dvsa/mot/trade/read/core/TradeReadServiceDatabase.java

```diff
@@ -130,12 +130,13 @@ public TradeReadServiceDatabase(MotTestReadService motTestReadService, VehicleRe
     public uk.gov.dvsa.mot.trade.api.Vehicle getLatestMotTestByRegistration(String registration) {
```

Comment:
method name is suggesting that is returning test, but actually, it is returning vehicle
is it something what can be easily adjusted? (what are side effects of changing method name?)

Reply:
This is the naming pattern used throughout the code. The Vehicle object encapsulates the latest MOT for the vehicle. It would be nice to change this approach at some point.